README.md

```diff
@@ -21,10 +21,11 @@ See the ["Rich text editor component for React"](https://ckeditor.com/docs/ckedi
 After cloning this repository, install necessary dependencies:
 
 ```bash
-npm install
+pnpm install
 ```
 
-You can also use [Yarn](https://yarnpkg.com/).
+> [!NOTE]
+> This project requires **pnpm v10** or higher. You can check your version with `pnpm --version` and update if needed with `npm install -g pnpm@latest`.
```

Comment:
Section about `pnpm` requirement should be placed above the `pnpm install` command.

Also, `README.md` contains `npm run ...` commands which now should be updated to `pnpm run ...`.